### PR TITLE
Restore google page

### PR DESCRIPTION
**Pull request by goshakri**

### CODE REVIEW

**Comment by goshakri** [comment]
[REFACT: Restore google](https://trello.com/c/zBRJ0tpG/23-refact-restore-google)

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,90 @@
...
+
+.random__search__btn:hover {
+  border: 1px solid rgba(0, 0, 0, 0.226);
+}
+
+.google__text {
+  justify-self: center;
+}
+
+.advertising {
```
Comment (hsher) [inline src/App.css:74]: Classes without styles are unused.
Remove unused items from css and html

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,72 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
```
Comment (hsher) [inline src/App.js:2]: This is atavism.
In our project we do not use this logo.
Remove logo.svg and remove import

File: src/stylesheets/base/_global.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  height: 100%;
+  margin: 0;
+  min-height: 100%;
+}
```
Comment (hsher) [inline src/stylesheets/base/_global.scss:18]: No newline at end of file.

File: src/stylesheets/modules/_MainWrapper.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.MainWrapper {
+  height: 100%;
+}
```
Comment (hsher) [inline src/stylesheets/modules/_MainWrapper.scss:3]: Check your Sublime Text 3 settings. It should add empty line by saving

File: src/App.js [hunk 5]
```diff
@@ -1,24 +1,71 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
+      </div>
+      <div className="Page-footer">
+        <a href="#" className="advertising">
```
Comment (hsher) [inline src/App.js:50]: Clean up unused classes

**Pull request merged**